sp/src/game/server/hl2/hl2_player.cpp

```diff
@@ -241,10 +241,9 @@ class CLogicPlayerProxy : public CLogicalEntity
 	void InputEnableCappedPhysicsDamage( inputdata_t &inputdata );
 	void InputDisableCappedPhysicsDamage( inputdata_t &inputdata );
 	void InputSetLocatorTargetEntity( inputdata_t &inputdata );
-#ifdef PORTAL
```

Comment:
I don't think removing all evidence that this was originally from Portal is wise, but I can add comments about it myself later.